addons/website_crm_partner_assign/views/website_crm_partner_assign_templates.xml

```diff
@@ -19,153 +17,241 @@
+                    </t>
+                </a>
+                <div class="dropdown-menu">
+                    <t t-foreach="grades" t-as="grade">
                         <a t-attf-href="/partners#{ grade['grade_id'][0] and '/grade/%s' % grade['grade_id'][0] or '' }#{ current_country and '/country/%s' % slug(current_country) or '' }#{ '?' + (search_path or '') + '&amp;' + keep_query('country_all') }"
-                           t-attf-class="nav-link#{grade['active'] and ' active' or ''}">
-                            <span class="badge rounded-pill float-end" t-esc="grade['grade_id_count'] or ''"/>
-                            <t t-esc="grade['grade_id'][1]"/>
-                        </a>
-                    </li>
-                </t>
-            </ul>
-
-            <ul id="reseller_countries" class="nav nav-pills flex-column mt16">
-                <li class="nav-header nav-item"><h3>Filter by Country</h3></li>
-                <t t-foreach="countries" t-as="country">
-                    <li t-if="country['country_id']" class="nav-item">
+                        class="dropdown-item" t-out="grade['grade_id'][1]"/>
+                    </t>
+                </div>
+            </div>
+            <div id="o_wcrm_reseller_countries" class="dropdown d-none d-lg-block">
+                <a role="button" href="#" data-bs-toggle="dropdown" class="dropdown-toggle btn btn-light" aria-expanded="true" aria-label="Open countries dropdown">
+                    <t t-foreach="countries" t-as="country">
+                        <span t-if="country['active']" t-out="country['country_id'][1]"/>
+                    </t>
+                </a>
+                <div class="dropdown-menu">
+                    <t t-foreach="countries" t-as="country" t-if="country['country_id']">
                         <a t-attf-href="/partners#{ current_grade and '/grade/%s' % slug(current_grade) or ''}#{country['country_id'][0] and '/country/%s' % country['country_id'][0] or '' }#{ '?' + (search_path or '') + (country['country_id'][0] == 0 and '&amp;country_all=True' or '')}"
-                           t-attf-class="nav-link#{country['active'] and ' active' or ''}">
-                            <span class="badge rounded-pill float-end" t-esc="country['country_id_count'] or ''"/>
-                            <t t-esc="country['country_id'][1]"/>
-                        </a>
-                    </li>
-                </t>
-            </ul>
-
-        </div>
-
-        <div class="col-lg-8 offset-lg-1" id="ref_content">
-            <div class="d-flex p-2">
-                <t t-call="website.pager"/>
-                <form action="" method="get" class="ms-auto">
-                    <div class="mb-3">
+                        class="dropdown-item" t-out="country['country_id'][1]"/>
+                    </t>
+                </div>
+            </div>
+            <div class="d-flex">
```

Comment:
On mobile, I'd not mind to have the search-box filling the entire available space.
![image](https://user-images.githubusercontent.com/8558351/231209536-5c71a7f3-bfd7-4601-9eca-2368bca66390.png)


Something like this should work:
```xml
<div class="d-flex w-100 w-lg-auto">
    <form class="flex-grow-1" action="" method="get">
    [...]
</div>
```